src/StatusQ/Components/StatusChatListItem.qml

```diff
@@ -28,8 +28,9 @@ Rectangle {
         width: 24
         height: 24
         color: Theme.palette.miscColor5
-        letterSize: emoji ? 14 : 15
+        letterSize: emoji ? 10 : 15
         emoji: ""
+        emojiSize: "16x16"
```

Comment:
Now that we get more emoji related properties, it'd be cool to have them similar to `icon` and `image` properties so that we could do: `emoji.name: ""`, `emoji.size: ...` etc.

I'll let you decide if we want to do this as part of this PR or keep it for the future because it'd be a breaking change.

Reply:
Good idea. Like you said though, it would be a breaking change, so I'll wait until the wallet and chat changes have settled so I don't conflict with concurrent changes.